DriverLoader.h

```diff
@@ -40,14 +41,26 @@
 namespace osvr {
 namespace vive {
     struct CouldNotLoadDriverModule : std::runtime_error {
-        CouldNotLoadDriverModule()
```

Comment:
I actually meant leave both constructors - a default constructor and one that takes an extra error message, with no platform ifdefs in this header at all...